src/operators.jl

```diff
@@ -851,3 +851,74 @@ function merge_vertices!(g::Graph{T}, vs::Vector{U} where U <: Integer) where T
 
     return new_vertex_ids
 end
+
+"""
+    mycielski(g)
+
+Returns a graph after applying the Mycielski operator to the input. The Mycielski operator
+returns a new graph with `2n+1` vertices and `3e+n` edges and will increase the chromatic
```

Comment:
I think using the symbol `e` for the number of edges is a bit unusual, when we use `n` for the number of vertices at the same time. Maybe there is  a better way?